README.md

```diff
@@ -3,14 +3,15 @@
 
 This repository contains the code for a persistent cache holding customs related data.
 
-| Path                               | Description                                          |
-| ---------------------------------  | ---------------------------------------------------- |
-| GET /customs-data-store/eori/:eori/verified-email | Retrieve the verified email address for a given EORI either from the cache or SUB09|
-| GET /customs-data-store/eori/:eori/company-information | Retrieves the business full name and address for the given EORI |
-| GET /customs-data-store/eori/:eori/eori-history | Retrieves the historic eori's for a given EORI either from the cache or SUB21        |
+| Path                               | Description                                                                               |
+| ---------------------------------  |-------------------------------------------------------------------------------------------|
+| GET /customs-data-store/eori/:eori/verified-email | Retrieve the verified email address for a given EORI either from the cache or SUB09       |
+| GET /customs-data-store/eori/:eori/company-information | Retrieves the business full name and address for the given EORI                           |
+| GET /customs-data-store/eori/:eori/eori-history | Retrieves the historic eori's for a given EORI either from the cache or SUB21             |
+| GET /customs-data-store/eori/xieori-information | Retrieves the XI EORI information for the requested EORI either from the cache or SUB09   |
 | POST /customs-data-store/update-email | Populates a new verified email address in the cache and removes undeliverable information | 
-| POST /customs-data-store/update-eori-history | Updates the eori history for a given EORI in the cache |
-| POST /update-undeliverable-email | Updates undeliverable information for a given enrolmentValue |
+| POST /customs-data-store/update-eori-history | Updates the eori history for a given EORI in the cache                                    |
+| POST /update-undeliverable-email | Updates undeliverable information for a given enrolmentValue                              |
 
 ## GET /eori/:eori/verified-email
 
```

Comment:
Can we add a space on line 37 to fix the format of the get company information?

currently. : ##GET /eori/:eori/company-information
should be: ## GET /eori/:eori/company-information

Reply:
Adam, its not part of this change but I have done it now.